data-validation/multiple-coordinate-schemas/policy.json

```diff
@@ -11,10 +11,10 @@
           "strategy": "ANY_OF",
           "schemas": [
             {
-              "schemaId": "europe-location"
+              "schemaId": "europe-coordinates"
             },
             {
-              "schemaId": "usa-location"
+              "schemaId": "usa-coordinates"
```

Comment:
Were these wrong?

Reply:
Yes, but there is another PR that will be merged that includes that fix, but anyhow will rebase it.